dot/parachain/network-bridge/sender.go

```diff
@@ -104,3 +107,48 @@ func (nbs *NetworkBridgeSender) processMessage(msg any) error {
 
 	return nil
 }
+
+const requestTimeout = 200 * time.Millisecond // TODO is this reasonable?
```

Comment:
maybe 2 - 5 seconds? I don't think such small timeout is beneficial, we might face many cancelations due to timeout